src/Sylius/Behat/Resources/config/suites/ui/admin/panel.yml

```
@@ -0,0 +1,27 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_panel:
+            contexts:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.transform.admin
+                - sylius.behat.context.transform.channel
+                - sylius.behat.context.transform.product
+                - sylius.behat.context.transform.taxon
+
+                - sylius.behat.context.setup.admin_user
+                - sylius.behat.context.setup.channel
+                - sylius.behat.context.setup.product
+                - sylius.behat.context.setup.admin_security
+                - sylius.behat.context.setup.taxonomy
```

```suggestion
                - sylius.behat.context.setup.admin_security
                - sylius.behat.context.setup.admin_user
                - sylius.behat.context.setup.channel
                - sylius.behat.context.setup.product
                - sylius.behat.context.setup.taxonomy
```
alphabetical order :v: